sky/skylet/job_lib.py

```diff
@@ -239,11 +261,14 @@ def _get_jobs_by_ids(job_ids: List[int]) -> List[Dict[str, Any]]:
     return records
 
 
-def query_job_status(job_owner: str, job_ids: List[int]) -> List[JobStatus]:
-    """Return the statuses of the jobs.
+def update_job_status(job_owner: str,
+                      job_ids: List[int],
+                      need_output: bool = True) -> List[JobStatus]:
```

Comment:
nit: `need_output` -> `silent`? The former suggests whether there's any returned value.